### PR TITLE
mui: update layout for tabs, simple list, simple show

### PR DESCRIPTION
**Pull request by killui**
Review a partir de second commit

### CODE REVIEW

File: src/mui/detail/SimpleShowLayout.js [hunk 1]
```diff
@@ -27,31 +26,27 @@ const SimpleShowLayout = ({
...
-                    ) : typeof field.type === 'string' ? (
-                        field
-                    ) : (
-                        React.cloneElement(field, {
-                            record,
-                            resource,
-                            basePath,
-                        })
-                    )}
+                <div>
```
Comment (Claire-Cleany) [inline src/mui/detail/SimpleShowLayout.js]: Don't you get a warning having removed the key in mapped element?

File: src/mui/list/InfiniteList.js [hunk 2]
```diff
@@ -19,6 +20,37 @@ import LinearProgress from 'material-ui/LinearProgress';
 import { List, mapStateToProps } from './List';
 import { defaultStyles } from '../defaultStyles';
 
+const InfiniteListTitle = ({ styles, titleElement, filters, actions, ...props }) => {
```
Comment (Claire-Cleany) [inline src/mui/list/InfiniteList.js]: You aren't using it, isn't it suppose to replaced a part of InfitineList?

File: src/mui/detail/Tab.js [hunk 3]
```diff
@@ -1,34 +1,32 @@
...
-                            React.cloneElement(field, rest)
-                        )}
-                    </div>
-                ) : null
-        )}
-    </span>
-);
+const Tab = ({ label, icon, children, ...rest }) => {
+  return (
+    <div>
```
Comment (thomas-cleany) [inline src/mui/detail/Tab.js]: same for intendation admin on rest expect 4

File: src/mui/detail/TabbedShowLayout.js [hunk 4]
```diff
@@ -24,36 +24,38 @@ class TabbedShowLayout extends Component {
...
-                                </Tab>
-                            ) : null
-                    )}
-                </Tabs>
-            </div>
+            <Tabs
+                value={this.state.value}
+                onChange={this.handleChange}
+                tabItemContainerStyle={styles.tabs}
+                inkBarStyle={styles.inkBar}
```
Comment (thomas-cleany) [inline src/mui/detail/TabbedShowLayout.js:51]: same for styles. this should be documented if someone wants to apply custom style to the Tabs

File: src/mui/list/SimpleList.js [hunk 5]
```diff
@@ -6,23 +6,26 @@ import { Link } from 'react-router-dom';
...
-    secondaryTextLines,
-    tertiaryText,
-    leftAvatar,
-    leftIcon,
-    rightAvatar,
-    rightIcon,
-    itemStyle,
-    itemInnerDivStyle,
-    disabled,
+  ids,
```
Comment (thomas-cleany) [inline src/mui/list/SimpleList.js]: same for indentation. You should do a make prettier :)

File: src/mui/detail/TabbedShowLayout.js [hunk 6]
```diff
@@ -2,7 +2,26 @@ import React, { Component } from 'react';
 import PropTypes from 'prop-types';
 import { Tabs, Tab } from 'material-ui/Tabs';
 
-const divStyle = { padding: '0 1em 1em 1em' };
+
+/**
+ *
+  styles: {
+    tabs: {
+      Style for container for the tabs
```
Comment (thomas-cleany) [inline src/mui/detail/TabbedShowLayout.js]: Style for tabs container

File: src/mui/detail/TabbedShowLayout.js [hunk 7]
```diff
@@ -2,7 +2,26 @@ import React, { Component } from 'react';
...
+    },
+    inkBar: {
+      Style for showing when a tab is active
+    },
+    activeButton: {
+      Style for Tab active
```
Comment (thomas-cleany) [inline src/mui/detail/TabbedShowLayout.js]: style for active Tab

File: src/mui/detail/TabbedShowLayout.js [hunk 8]
```diff
@@ -2,7 +2,26 @@ import React, { Component } from 'react';
...
+    },
+    button: {
+      Style for Tab inactive
```
Comment (thomas-cleany) [inline src/mui/detail/TabbedShowLayout.js]: style for inactive Tab

File: src/mui/detail/TabbedShowLayout.js [hunk 9]
```diff
@@ -2,7 +2,26 @@ import React, { Component } from 'react';
...
-const divStyle = { padding: '0 1em 1em 1em' };
+
+/**
+ *
+  styles: {
+    tabs: {
+      Style for container for the tabs
+    },
+    inkBar: {
+      Style for showing when a tab is active
```
Comment (thomas-cleany) [inline src/mui/detail/TabbedShowLayout.js]: That one is not very clear and confusing with style.activeButton

**Review by thomas-cleany — APPROVED** [review]

**Review by Claire-Cleany — APPROVED** [review]

**Pull request merged**